README.md

```diff
@@ -21,8 +21,17 @@ composer require escopecz/mautic-form-submit
 ## Usage
 
 ```php
+//basic form data
+$email		= $_REQUEST['email'] ?? $_REQUEST['Email'] ?? $_REQUEST['E-mail'] ?? $_REQUEST['E-MAIL']; // mauticform_input_mkregistraciaslp_email
+$name		  = $_REQUEST['name'] ?? $_REQUEST['Name']; // mauticform_input_mkregistraciaslp_name
+$phone		= $_REQUEST['phone'] ?? $_REQUEST['Phone']; // mauticform_input_mkregistraciaslp_phone
+$form_id	= $_REQUEST['form_id'] ?? $_REQUEST['Form_id']; // mauticform_input_mkregistraciaslp_form_id
+if (is_null($form_id)) {$form_id  = 1;} // if form_id is empty then use the 1'st Mautic form
+$form   = $mautic->getForm($form_id);
+
 // Require Composer autoloader
 require __DIR__.'/vendor/autoload.php';
+// or  require __DIR__.'/../../vendor/autoload.php'; if you place it not in the root folder, but for example in the /docroot/webhook/ folder
```

Comment:
Isn't this obvious? The autoloader path must be correct and depends on where the vendor dir is relative to this PHP script. It may be different for everyone. Copy-pasting code and hoping it will work without understanding it will be a big issue now with ChatGPT, I know. But even the young developers must thing about what the example code does.

Reply:
This is obvious to you, a senior programmer, but it is not obvious to a marketer who needs to send data to mautic from a website.